include/xtensor/xsort.hpp

```diff
@@ -266,30 +266,76 @@ namespace xt
         );
     }
 
+    /*****************************
+     * Implementation of argsort *
+     *****************************/
+
+    /**
+     * Sorting method.
+     *
+     * Predefined methods for performing indirect sorting.
+     *
+     * @see argsort(const xexpression<E>& e, std::ptrdiff_t axis = -1, sorting_method method)
```

Comment:
I think this can be simply
```suggestion
     * @see argsort(const xexpression<E>&, std::ptrdiff_t, sorting_method)
```